external-crates/move/crates/move-bytecode-source-map/src/mapping.rs

```diff
@@ -36,9 +36,13 @@ impl<'a> SourceMapping<'a> {
         }
     }
 
-    pub fn new_without_source_map(bytecode: &'a CompiledModule, default_loc: Loc) -> Result<Self> {
+    pub fn new_without_source_map(
+        bytecode: &'a CompiledModule,
+        default_loc: Loc,
+        default_definition_loc: Loc,
```

Comment:
I would remove this, and just use `default_loc` in all places. This is passing a default location for everything so passing multiple default locs feels wrong.

This would also make the changes a lot smaller/more localized as well as then `from_module` would stay the same as it is today as well.

Reply:
Done!